jerry-core/debugger/debugger.c

```diff
@@ -202,6 +218,23 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
 } /* jerry_debugger_send_eval */
 
 /**
+ * Suspend execution for received time
+ */
+static void
+jerry_debugger_sleep (int milliseconds) /**< Suspending time */
+{
+#ifdef HAVE_TIME_H /* HAVE_TIME */
+  nanosleep (&(const struct timespec)
+  {
+    milliseconds / 1000, milliseconds * 1000000 /* Seconds, nanoseconds */
```

Comment:
math is still not good. e.g., in case of 1500 millisecs, this will be 1 sec + 1500 * 10^6 nanosecs (where 500 * 10^6 wuld be correct)

one more note: milliseconds is int, and multiplying it by literal 1000000 will also yield an int, which may overflow on some architectures (although improbable on 32 bit systems). using a long literal (1000000L) can solve the problem for sure (and the second field of the timespec struct is long anyway).